docs/src/main/sphinx/release/release-376.md

```diff
@@ -0,0 +1,83 @@
+# Release 376 (7 Apr 2022)
+
+## General
+
+* Add table redirection awareness for `RENAME table` operations. ({issue}`11277`)
+* Deny adding column with comment if the connector does not support this feature. ({issue}`11486`)
+* Improve query plans and performance in the presence of inequality expressions. ({issue}`11518`)
+* Use "is distinct" semantics in `array_except`, `array_union`, `map_concat`,
```

Comment:
Minor rewording here from your proposed text @jirassimok .. fyi @losipiuk .. let me know if this is ok

Reply:
This should be rephrased in terms of the effect on those functions. Maybe something like: "consider NULLs as identical values in xxx functions".

Reply:
Adapted